static-libraries/js/jquery.cookie.js

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
It is used in dyna-tree persist variable.
the persist attribute require this file to generate a jquery cookie.